client/platform/web-girder/views/Home.vue

```diff
@@ -157,12 +180,53 @@ export default Vue.extend({
       height="6"
       :style="{ visibility: loading ? 'visible' : 'hidden' }"
     />
-    <v-container fill-height>
+    <v-container
+      fill-height
+      :fluid="$vuetify.breakpoint.mdAndDown"
+    >
       <v-row
         class="fill-height nowraptable"
-        no-gutters
       >
-        <v-col :cols="12">
+        <v-col cols="3">
+          <DataDetails
```

Comment:
scroll sticky?  I know that isn't something that the file browser has so it isn't really that important.
![image](https://user-images.githubusercontent.com/61746913/113619135-98b73800-9626-11eb-8930-373a3e44b4c6.png)

Reply:
I wasn't really feeling this one. You're welcome to add it